test/framework/imageregistry.go

```diff
@@ -154,7 +154,16 @@ func RemoveImageRegistry(logger Logger, client *Clientset) error {
 }
 
 func MustRemoveImageRegistry(t *testing.T, client *Clientset) {
-	if err := RemoveImageRegistry(t, client); err != nil {
+	err := wait.Poll(5*time.Second, AsyncOperationTimeout, func() (bool, error) {
+		if err := RemoveImageRegistry(t, client); err != nil {
```

Comment:
just realized removeimageregistry doesn't return the client error directly, need to rework this a bit.

Reply:
ok updated.